src/index.html

```diff
@@ -1,14 +1,67 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+    href="https://fonts.googleapis.com">
```

Comment:
You need to check and fix it everywhere
<img width="1048" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/9f7c2e5c-ed35-42c9-8542-325c71e06af6">

Reply:
didn't fix it https://github.com/mate-academy/layout_moyo-header/pull/3390/files#r1251855817